packages/core/src/external/sde/file-names.ts

```diff
@@ -0,0 +1,74 @@
+import { MetriportError } from "@metriport/shared";
+
+export function getCdaToFhirConversionPrefix({
+  cxId,
+  patientId,
+}: {
+  cxId: string;
+  patientId: string;
+}): string {
+  return `${cxId}/${patientId}/${cxId}_${patientId}_`;
+}
+
+export function buildDocumentConversionFileName({
+  cxId,
+  patientId,
+  documentId,
+  extension = ".xml.json",
+}: {
+  cxId: string;
+  patientId: string;
+  documentId: string;
+  extension?: string;
+}): string {
+  return `${cxId}/${patientId}/${cxId}_${patientId}_${documentId}${extension}`;
+}
+
+export function parseCdaToFhirConversionFileName({ fileName }: { fileName: string }): {
+  cxId: string;
+  patientId: string;
+  documentId: string;
+  extension?: string | undefined;
+} {
+  const [cxId, patientId, documentFileName] = fileName.split("/");
+  if (!documentFileName) {
+    throw new MetriportError(`Invalid cda to fhir conversion file name: ${fileName}`);
```

Comment:
Static error messages, please. Can add the filename as third param to MetriportError